test/search/search.test.js

```diff
@@ -101,7 +104,37 @@ describe('Search Tests', () => {
       expect(response).to.have.status(200);
       expect(response.body).to.be.an('object');
       expect(response.body.data.count).to.equal(0);
-      expect(response.body.itemsOnPage).to.equal(0);
+    });
+  });
+
+  context('when the user enters a global search query', () => {
+    it('returns the search results', async () => {
+      const response = await chai
+        .request(app)
+        .get(`${BASE_URL}/search/?global=j`);
+      expect(response).to.have.status(200);
+      expect(response.body).to.be.an('object');
+      expect(response.body.userSearch.count).to.not.equal('0');
+    });
+  });
+
+  context('when the user enters a article search query', () => {
+    it('returns the search results', async () => {
```

Comment:
Can this test tell us more about the structure of the search result.

Reply:
done